charts/flyte-binary/templates/deployment.yaml

```diff
@@ -192,15 +192,17 @@ spec:
           {{- else }}
             httpGet:
               path: /healthcheck
-              port: http
+              port: 8088
```

Comment:
@guyarad Could you reproduce this by reverting to using the named port? I have a Flyte env on EKS using Public API Server and public subnets and I can't reproduce the `livenessProbe` failure with the named port